bin/cal

```diff
@@ -16,47 +16,25 @@ package
 	cal; # hide from PAUSE
```

Comment:
We could probably get rid of the special package too. Not a big deal or necessary for this.